socrata/socrata_upload.py

```diff
@@ -0,0 +1,174 @@
+import requests
+import os
+from pyathena import connect
+from pyathena.pandas.util import as_pandas
+from pyathena.pandas.cursor import PandasCursor
+from dotenv import load_dotenv
+
+# Load environmental variables and connect to Athena
+load_dotenv(".Renviron")
+app_token = os.getenv("SOCRATA_APP_TOKEN")
+auth = (os.getenv("SOCRATA_USERNAME"), os.getenv("SOCRATA_PASSWORD"))
+cursor = connect(
+    s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR") + "/",
+    region_name=os.getenv("AWS_REGION"),
+    cursor_class=PandasCursor,
+).cursor(unload=True)
+
+
+def build_query(athena_asset, row_identifiers, years=None):
```

Comment:
@dfsnow is this something as user should pass to the function, or something we could automatically grab from dbt